### DISCUSSION TITLE
[Design/Antagonist] Zombie outbreak

### DISCUSSION THREAD

**Discussion by Peakwastaken** [ParadiseSS13/Paradise#23131] [Documents]
I've been rummaging on how to make a zombie outbreak as a mid-round a fun major event for the station, here's what I came up with :

**The outbreak :**
The event would start with one-three "Alpha Zombies" (edgy name subject to change) who are pooled from ghosts, spawned randomly in maint, and stronger in many ways to regular zombies (details below). They'll be the prime vector of the zombie outbreak, dolling out the first wave of infections.

Edit : Alpha zombies could also check for non-revivable bodies (DNR/suicides) to spawn as.

The infection starts with a disease which takes 3.5 minutes to reach maturity (doesn't affect IPC) : 
- First stage lasts two and a half minutes - no symptoms, shows up on body scanner (not hud/healthanalyzer) but has no known cure.
- Second stage lasts the remainder of the one minute  - symptoms are : confusion, blood vomiting, projectile vomiting, necrotizing fasciitis and itching. Some on-brand kinda stuff
- Third stage the disease "cures" itself and instead spawns a tumor in the host's torso, that heals them to full and makes them turn into a zombie giving them a notice of their new antag status. \

Edit : if the infected dies at any stage of the infection, it skips directly to the tumor step and starts the reviving cooldown.

The tumor is basically what makes them be a zombie, removing it, or having the brain/head of the host be transplanted onto a new body will cure it. Removing the tumor out of an alpha zombie makes it gib/dust or any other way to make them unrevivable.

If an infected person's blood (while they're still within the "diseased" stage and before the tumor shows up) is put in a pandemic machine, it starts a 5 minutes timer after which it'll produce a cure which can be replicated and given out to the crew to prevent further infection (but it won't cure current zombies who need their tumors removed).

Edit : Zombies always show up dead on medhuds, (maybe not on examine though) allowing them to play dead, as well as helping identify zombies who are running around in fully covering clothes like modsuits.

**The Zombies :**

Alpha Zombies are naked random race carbons (or a custom sprite if people are interested), can't wear clothes and have the following traits :

- They are a dark shade of green, and glowing red eyes.
- They have two no-drops in hand items that represent a claw and a set of teeth for clawing/biting. Clawing makes them do 20-25 brute and lets them bash or pry open doors like jol/fireaxe, biting does only 15 brute but infects people, and doing so heals 15 brute/burn. This is both to give them weapons that can break doors/windows and prevents them from using anything a regular carbon could use. Clawing could make the sound faithlesses do, biting terror's spider sound and animation. (No armor penetration. 10-12.5/7.5 brute on riot armor)
- They revive upon death with a full aheal akin to a cling after 30 seconds unless they're beheaded/gibbed/incinerated/their tumor has been removed.
- they ignore slowdown, don't require oxygen, and can't be stunned.
- They regenerate -4 brute/burn/toxins per cycle, twice salglu healing rate + toxins.
- Their speech pattern is changed into gurgle (and/or with random shouts as they hit people like hulk) to non-zombies, letting them give orders/coordinate with nearby alpha/regular zeds.
- They can leap like the genetic power with twice the cooldown as long as they're not bola'd (which should apply to the jump power in general frankly).

Basic Zombies are regular carbon whatever species of the person who got turned and their gear but : 

- They are a lighter shade of green.
- Clawing makes them do 15-21 brute (barely enough to slowly wittle airlocks) and lets them pry open doors like jol/fireaxe, biting does only 10 brute but infects people, and makes the zombie heal 10 brute/burn upon doing so. (7.5-10.5/5 brute on riot armor)
- They revive upon death with a full aheal after a minute unless they're beheaded//gibbed/incinerated/their tumor has been removed.
- They ignore slowdown, don't require oxygen, and can't be stunned.
- They regenerate -2 brute/burn/toxins per cycle, like salglu would + toxins.
- Their speech pattern is changed into gurgles (and/or with random shouts as they hit people like hulk).

**Why I think it'd work and extra tidbits :**

Having ghost roles be part of the infection is fun for observers, and it having both a disease/tumor phase gives a lot of flexibility for incorporation into events, as well as multiple ways to deal with the outbreak depending on the station's state of disarray.

First you might just decapitate bodies and preserve them while you work on getting the tumor out, reattach the heads to new bodies, or run the risk of removing a tumor on a potentially soon reviving zombie. You can also quarantine people who've been bit but survived their encounter with the zeds. Then when the cure is out, vaccinated doctors can operate more easily on zombie without decapitating them while having armed supervision.

Decapitating all fallen zombies might seem like an easy way out but it might be very risky to get so close to one before you have the vaccine, and you'll quickly widdle out able bodies to fight back against the outbreak. And them being carbons mean that, first - people keep their characters, bolas and aiming to cripple the legs are very effective ways to deal with the zombies, as well as using defenses like sandbags or tables.

The vaccine will take some time to develop even after a blood sample has been provided, giving time for the infection to take place a little more. If the outbreak is thwarted early, there'll be no or very little damage to the station/crew. If it takes a while, the damage could be interesting, bring a lot of RP, and change the round's flow without removing people permanently (I'm thinking a full crew of different species' heads on human bodies, sounds like peak ss13 to me !). If it isn't containable, though, it'll be a glorious fight for survival akin to massive xeno/t-spider outbreak, worthy of a nuke or DS.

I'd love to hear what you guys think about this, I've been thinking of maybe having armor/full coverage reduce the chance of bites contaminating the victim, but I'm not sure if it's necessary as I've tried to make it as balanced as my mind can project, numbers would likely have to be tested and tweaked.

I've also tried to keep it to mechanics that are somewhat in the game (white t-spider infestation, tumors, vampiric claws, cling rejuv (which it could literally be that), the jump power, very little to sprite) so that it can be implemented as easily as possible.

**Comment by bakaf** [comment]
Sounds good, I'd never say no to zombies

**Comment by Skycert** [comment]
I feel you should read this (if you haven't): https://www.paradisestation.org/forum/topic/23186-zombies-design-document

I think your idea is cooler, but if I were to salvage something from mine it would be the "playing dead" thing. I would like to see zombies done, but I'm never enough un-busy to get on to it...

**Comment by Peakwastaken** [comment]
> I think your idea is cooler, but if I were to salvage something from mine it would be the "playing dead" thing. I would like to see zombies done, but I'm never enough un-busy to get on to it...

Ah yes, really like this, both for the playing dead part as well as to give a way to identify zombies who are fully covered like in modsuits and the like !

Updated the design doc with the suggestions given to me !
